src/engraving/rw/compat/compatutils.cpp

```diff
@@ -341,6 +342,78 @@ void CompatUtils::reconstructTypeOfCustomDynamics(MasterScore* score)
     }
 }
 
+void CompatUtils::splitArticulations(MasterScore* masterScore)
+{
+    std::set<Articulation*> toRemove;
+    for (Score* score : masterScore->scoreList()) {
+        for (Measure* meas = score->firstMeasure(); meas; meas = meas->nextMeasure()) {
+            for (Segment& seg : meas->segments()) {
+                if (!seg.isChordRestType()) {
+                    continue;
+                }
+                for (EngravingItem* item : seg.elist()) {
+                    if (!item || !item->isChord()) {
+                        continue;
+                    }
+                    Chord* chord = toChord(item);
+                    for (Articulation* a : chord->articulations()) {
+                        std::set<SymId> ids = mu::engraving::splitArticulations({ a->symId() });
+                        if (ids.size() <= 1) {
+                            continue;
+                        }
+                        toRemove.insert(a);
+                    }
+                }
+            }
+        }
+        // separate into individual articulations
+        for (Articulation* combinedArtic : toRemove) {
```

Comment:
I think either `toRemove` needs to be inside the `for (Score* score : masterScore->scoreList()) {` loop, _or_ `for (Articulation* combinedArtic : toRemove) {` needs to be outside that loop. Because now, for the second score, the items in the first score will be seen again.